<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KompozycjeDialog</class>
 <widget class="QDialog" name="KompozycjeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>COMPOSITIONS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/GIAP-giap_layout/icons/giap_logo.png</normaloff>:/plugins/GIAP-giap_layout/icons/giap_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: rgb(53, 85, 109);
    font: 10pt &quot;Segoe UI&quot;; 
	color: white;
}
QAbstractItemView
{
	alternate-background-color:  #e6e6e6;
}
QComboBox
{
	selection-background-color: #b7d3e8;
    background-color: #4f5a63;
	border-style: solid;
	border: 1px solid  #5689b0;
	border-radius: 3px;
	padding: 2px;
	font: 9pt &quot;Segoe UI&quot;;
    height: 20px;
}

QComboBox:on
{
	background-color: #4f5a63;
	padding-top: 3px;
	padding-left: 1px;
	selection-background-color: #4a4a4a;
}

QComboBox QAbstractItemView
{
	background-color: #4f5a63;
	border-radius: 4px;
	border: 1px solid  #6fb1e3;
	selection-background-color: #4f5a63;
    padding: 4px 10px 4px 10px;
    width: 1.9em;
}

QComboBox::drop-down
{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
    background-color: #5689b0;
	border-left-width: 0px;
	border-left-color:  #1a2936;
	border-left-style: solid;
	border-top-right-radius: 3px; 
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow
{
	image: url(icons/down_arrow_disabled.png);
}

QComboBox::down-arrow:on, 
QComboBox::down-arrow:hover,
QComboBox::down-arrow:focus
{
	image: url(icons/down_arrow.png);
}

QLineEdit
{
	border: 1px solid #5689b0;
    border-radius: 3px;
    background-color: #4f5a63;
}

QPushButton {
	color: black;
	border: 1px solid black;
	background-color: #b7d3e8;
	border-radius: 3px;
	padding: 4px;
}

QPushButton:hover {
	font-weight: bold;
}
QPushButton:pressed {
   background-color: #5689b0;
   border: 1px solid #6fb1e3;
}

QPushButton:disabled
{
	color: gray;
}

QGroupBox {
border: 1px solid #5689b0;
}

QToolButton {
   border: 0px solid transparent;
   color: black;

}

QToolButton:hover {
   background-color: #5689b0;
   border: 1px solid #6fb1e3;
}

QToolButton:pressed {
   background-color: #5689b0;
   border: 1px solid #6fb1e3;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="checkPushButton">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uncheckPushButton">
           <property name="text">
            <string>Unselect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="checkAllPushButton">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uncheckAllPushButton">
           <property name="text">
            <string>Unselect All</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" rowspan="5">
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
border-radius: 10px;
background-color: white;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px;
border-radius: 0px;
border-bottom: 1px solid black;
color: black;</string>
           </property>
           <property name="text">
            <string>Groups</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="grupy_table">
           <property name="styleSheet">
            <string notr="true">QListView {
border: 0px;
color: black;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QToolButton" name="wdol_warstwe">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::DownArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wgore_warstwe">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::UpArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Composition name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nazwa_lineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2" rowspan="4">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="dodaj_warstwe">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="usun_warstwe">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="5">
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
border-radius: 10px;
background-color: white;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px;
border-radius: 0px;
border-bottom: 1px solid black;
color: black;</string>
           </property>
           <property name="text">
            <string>Layers in Groups</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="warstwy_w_grupie_table">
           <property name="styleSheet">
            <string notr="true">QListView {
border: 0px;
color: black;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QFrame" name="frame_5">
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
border-radius: 10px;
background-color: white;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 0px;
border-radius: 0px;
border-bottom: 1px solid black;
color: black;</string>
           </property>
           <property name="text">
            <string>Selected Layers</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="warstwy_table">
           <property name="styleSheet">
            <string notr="true">QListView {
border: 0px;
color: black;
}</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::NoDragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #4f5a63;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="title_label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>261</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color : white;	font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Add new composition</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>8</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
* {
	font: 75 8pt &quot;Segoe UI Light&quot;;
	color: black;
}

QFrame {
	border-right: 2px solid #b7d3e8;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">   background-color: #b7d3e8;
   color: black;
   font: 10pt &quot;Segoe UI&quot;; 
   font-weight: bold;
   padding-left: 5px;
   padding-right: 5px;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nazwa_lineEdit</tabstop>
  <tabstop>dodaj_warstwe</tabstop>
  <tabstop>checkPushButton</tabstop>
  <tabstop>uncheckPushButton</tabstop>
  <tabstop>checkAllPushButton</tabstop>
  <tabstop>uncheckAllPushButton</tabstop>
  <tabstop>usun_warstwe</tabstop>
  <tabstop>wdol_warstwe</tabstop>
  <tabstop>wgore_warstwe</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>KompozycjeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1227</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
